backend/creditsWorker.js

```diff
@@ -0,0 +1,190 @@
+'use strict'
+
+import sbp from '@sbp/sbp'
+import { readyQueueName } from './genericWorker.js'
+
+const TASK_TIME_INTERVAL = 300e3
+
+// Rate: How many credits are charged per byte stored per second.
+// Using BigInt for precision.
+// $FlowFixMe[cannot-resolve-name]
+const CREDITS_PER_BYTESECOND = BigInt(1)
```

Comment:
Hmm... are you sure every database we want to support can handle this `BigInt`?

Reply:
Yes, `BigInt` values are arbitrary precision integers in JS. Using them on a DB will usually require converting to a string first (or converting from a string), even if the DB also happens to have a dedicated `BigInt` type. The reason for using `BigInt` instead of `number` is precision, since float types computations are inexact (except in some cases) and errors carry over. Generally, it's not recommended to use float for anything currency or money-related or -adjacent, which is why this uses `BigInt`.